src/main/java/mate/academy/spring/dao/UserDaoImpl.java

```diff
@@ -6,12 +6,15 @@
 import org.hibernate.Session;
 import org.hibernate.SessionFactory;
 import org.hibernate.Transaction;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public class UserDaoImpl implements UserDao {
+
     private final SessionFactory sessionFactory;
 
+    @Autowired
     public UserDaoImpl(SessionFactory sessionFactory) {
```

Comment:
If you have one constructor @Autowired it's unnecessary. It's not mistake just information)